.github/workflows/ci.yml

```
@@ -71,13 +95,26 @@ jobs:
         continue-on-error: true
 
+      - name: Generate JaCoCo test coverage report
+        run: ./gradlew jacocoTestReport
+        if: always()
+
+      - name: Check test coverage (80%)
+        run: ./gradlew jacocoTestCoverageVerification || true
+        continue-on-error: true
+
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)